### PR TITLE
DCS-1665 bumping dependencies

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: backend/sessionManagementRoutes.js [hunk 1]
```diff
@@ -25,10 +25,13 @@ const configureRoutes = ({ app, tokenRefresher, tokenVerifier }) => {
     return passport.authenticate('oauth2')(req, res, next)
   }
 
-  const logout = (req, res) => {
-    req.session.destroy(() => {
-      res.redirect(authLogoutUrl)
-    })
+  const logout = (req, res, next) => {
+    if (req.user) {
+      req.logout(err => {
```
Comment (andrewrlee) [inline backend/sessionManagementRoutes.js:30]: logout now is async and needs a callback function

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ env:
   HMPPS_COOKIE_NAME: hmpps-session-dev
   REMOTE_AUTH_STRATEGY: "true"
   WEB_SESSION_TIMEOUT_IN_MINUTES: "60"
-  VIDEO_LINK_ENABLED_FOR: "WWI,TSI,HEI,BWI,NMI,EYI,BNI,PBI,BMI"
+  VIDEO_LINK_ENABLED_FOR: "WWI,TSI,HEI,BWI,NMI,EYI,BNI,PBI,BMI,NWI"
```
Comment (andrewrlee) [inline helm_deploy/values-dev.yaml:31]: adding norwich to dev and preprod in prep for future release

File: package.json [hunk 3]
```diff
@@ -9,78 +9,78 @@
...
-    "set-cookie-parser": "^2.4.8",
-    "supertest": "^6.2.2",
+    "sass": "^1.52.3",
+    "set-cookie-parser": "^2.5.0",
+    "supertest": "^6.2.3",
     "ts-jest": "^27.1.4",
-    "typescript": "^4.6.3"
+    "typescript": "^4.7.3"
   },
   "scripts": {
```
Comment (andrewrlee) [inline package.json:85]: this still needs the following updates, but they're quite large changes so spinning off into separate PRs: 

* redis
* cypress
* govuk frontend 

Also jest but that requires a defect in jest-html-reporter to be fixed first

**Review by connormaglynn — APPROVED** [review]
lgtm

**Pull request merged**